src/assets/scripts/client/measurement/MeasureTool.js

```diff
@@ -0,0 +1,513 @@
+import GameController from '../game/GameController';
+import EventBus from '../lib/EventBus';
+import { radians_normalize } from '../math/circle';
+import { bearingToPoint } from '../math/flightMath';
+import { distance2d } from '../math/distance';
+import {
+    heading_to_string,
+    km,
+    nm
+} from '../utilities/unitConverters';
+import MeasureLegModel from './MeasureLegModel';
+import AircraftModel from '../aircraft/AircraftModel';
+import FixModel from '../navigationLibrary/FixModel';
+import { EVENT } from '../constants/eventNames';
+import { GAME_OPTION_NAMES } from '../constants/gameOptionConstants';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * Style enumeration of available render styles
+ */
+export const MEASURE_TOOL_STYLE = {
+    STRAIGHT: 0,
+    INITIAL_TURN: 1,
+    ARCED: 2
+};
```

Comment:
I can see what you mean, but my only concern is that we'll be using string like `'straight'` in various places. Using the constants means they're only defined one.

I'm happy to go to string constants instead of bitwise operators, but wouldn't it make sense to keep `MEASURE_TOOL_STYLE` constant and import it into `gameOptionsConsts`?

``` js
            {
                displayLabel: 'Straight lines only',
                value: MEASURE_TOOL_STYLE.STRAIGHT
            },
```

Reply:
Fair point. And yes, that's a perfect solution then.